integration/mediation-tests/tests-mediator-1/src/test/java/org/wso2/carbon/esb/mediator/test/property/PropertyIntegrationAxis2ScopeRemovePropertiesTestCase.java

```diff
@@ -0,0 +1,145 @@
+    }
+
+    @Test(groups = "wso2.esb",
+          description = "Remove action as \"value\" and type Float (axis2 scope)")
+    public void testFloatVal() throws Exception {
+        logViewer.clearLogs();
+        OMElement response = axis2Client
+                .sendSimpleStockQuoteRequest(getProxyServiceURLHttp("propertyFloatAxis2RemoveTestProxy"), null, "Random Symbol");
+        assertTrue(response.toString().contains("Property Set and Removed"),
+                "Proxy Invocation Failed!");
+        assertTrue(isMatchFound("symbol = 123.123"),
+                "Float Property Not Either Set or Removed in the Axis2 scope!!");
+    }
+
+    @Test(groups = "wso2.esb",
+          description = "Remove action as \"value\" and type Long (axis2 scope)")
+    public void testLongVal() throws Exception {
+        logViewer.clearLogs();
+        OMElement response = axis2Client
+                .sendSimpleStockQuoteRequest(getProxyServiceURLHttp("propertyLongAxis2RemoveTestProxy"), null, "Random Symbol");
+        assertTrue(response.toString().contains("Property Set and Removed"),
+                "Proxy Invocation Failed!");
+        assertTrue(isMatchFound("symbol = 123"),
+                "Long Property Not Either Set or Removed in the Axis2 scope!!");
+    }
+
+    @Test(groups = "wso2.esb",
+          description = "Remove action as \"value\" and type Short (axis2 scope)")
+    public void testShortVal() throws Exception {
+        logViewer.clearLogs();
+        OMElement response = axis2Client
+                .sendSimpleStockQuoteRequest(getProxyServiceURLHttp("propertyShortAxis2RemoveTestProxy"), null, "Random Symbol");
+        assertTrue(response.toString().contains("Property Set and Removed"),
+                "Proxy Invocation Failed!");
+        assertTrue(isMatchFound("symbol = 12"),
+                "Short Property Not Either Set or Removed in the Axis2 scope!!");
+    }
+
+    @Test(groups = "wso2.esb",
+          description = "Remove action as \"value\" and type OM (axis2 scope)")
+    public void testOMVal() throws Exception {
+        logViewer.clearLogs();
+        OMElement response = axis2Client
+                .sendSimpleStockQuoteRequest(getProxyServiceURLHttp("propertyOMAxis2RemoveTestProxy"), null, "Random Symbol");
+        assertTrue(response.toString().contains("Property Set and Removed"),
+                "Proxy Invocation Failed!");
+        assertTrue(isMatchFound("symbol = OMMMMM"),
+                "OM Property Not Either Set or Removed in the Axis2 scope!!");
+    }
+
+    private boolean isMatchFound(String matchStr) throws Exception {
+        boolean isSet = false;
+        LogEvent[] logs = logViewer.getAllSystemLogs();
+        int size = logs.length;
+        for (int i = size-1; i >= 0; i--) {
+            if (logs[i].getMessage().contains(matchStr)) {
+                for (int j = i; j >= 0; j--) {
+                    if (logs[j].getMessage().contains("symbol = null")) {
```

Comment:
Can you explain the relevance of 'symbol = null' here ? shall we define this as a constant ? could you also include java docs to explaining the match process ?

Reply:
Added method comment in #2148